apps/aechannel/src/aesc_fsm.erl

```diff
@@ -382,6 +384,18 @@ error_code_to_msg(Code) when Code >= ?ERR_USER ->
 error_code_to_msg(_) ->
     <<"unknown">>.
 
+-spec stop(pid()) -> ok.
+stop(Fsm) ->
+    lager:debug("Gracefully stopping the FSM ~p", [Fsm]),
+    stop_ok(catch gen_statem:stop(Fsm, shutdown, 5000)).
+
+stop_ok(ok) ->
```

Comment:
Can you document why these clauses are handled specifically as ok?

Reply:
This function should be removed entirely as soon as it's removed from the `aehttp_sc_SUITE`